Server/Controllers/HomeController.fs

```diff
@@ -0,0 +1,24 @@
+namespace Codingteam.Site.Controllers
+
+open System
+open System.Globalization
+
+open Microsoft.AspNetCore.Mvc
+
+type HomeController() =
+    inherit Controller()
+
+    let logUrlPrefix = "http://0xd34df00d.me/logs/chat/codingteam@conference.jabber.ru"
```

Comment:
Should be configurable. We shouldn't rely on Mr Dead Food's server to be online and accessible by that hostname.